docs/scRNAseq_basics/import_exo.md

```diff
@@ -0,0 +1,45 @@
+![](images/yourmove.png){width="500"}
+
+Now it's your time to shine ! We are going to put into pratice what we 
+have just seen. By using a more complicated use case, you are going to 
+reproduce the whole scRNAseq analysis with Seurat.  
+
+# Dataset test
+
+The dataset for this analysis will be single cell RNAseq from chick embryos 
+from [de Lima *et al*](https://doi.org/10.1038/s41467-021-24157-x). We are 
+going to analyse only single cells of E10 stage of embryos. You can download 
+the dataset at the GEO accession [GSE166981](https://0-www-ncbi-nlm-nih-gov.brum.beds.ac.uk/geo/query/acc.cgi?acc=GSE166981).
+
+??? tip "Do you need help to find the data ?"
```

Comment:
C'est un peu dur de demander si on a besoin d'aide pour trouver le dataset, alors que tu n'explique ce que tu veux QUE dans le tip caché !!!

J'appelerais plutôt ça:

```
??? tip "Instructions to select the dataset(s)"
```

Reply:
Pas forcément d'accord puisque pour moi je le dis ici : `We are going to analyse only single cell of E10 stage of embryos.` mais bon si déjà on est pas clair c'est que c'est mal expliqué ! Je voulais qu'ils fouillent un peu GEO et je pense que certaines personnes savent deja où trouver les infos. Mais je me trompe peut etre. 
Sinon finalement faut dire directement les étapes. C'est ce que tu demandes ?

Reply:
Bon, j'ai l'impression que ce n'est pas très intuitif ce que tu demandes mais mergeons, là c'est devenu un peu trop complexe, j'ai besoin de voir la copie "propre"...

Reply:
ça marche en attendant je vais réfléchir à un moyen d'être plus clair et/ou plus simple.